packages/plugin-default-event-tracking-advanced-browser/src/default-event-tracking-advanced-plugin.ts

```diff
@@ -184,27 +178,29 @@ export const defaultEventTrackingAdvancedPlugin = (options: Options = {}): Brows
     }
     const addListener = (el: Element) => {
       if (shouldTrackEvent('click', el)) {
-        addEventListener(el, 'click', () => {
+        addEventListener(el, 'click', (event: Event) => {
           /* istanbul ignore next */
           amplitude?.track(constants.AMPLITUDE_ELEMENT_CLICKED_EVENT, getEventProperties('click', el));
+          event.stopPropagation();
```

Comment:
could this impact events listener added by the customer for some other purpose?

Reply:
Thanks for reviewing! Good question, it shouldn't, `stopPropagation` only prevents this event handler bubbling, there is another method [`stopImmediatePropagation`](https://developer.mozilla.org/en-US/docs/Web/API/Event/stopImmediatePropagation) preventing other event listeners:
> If several listeners are attached to the same element for the same event type, they are called in the order in which they were added. If stopImmediatePropagation() is invoked during one such call, no remaining listeners will be called, either on that element or any other element.